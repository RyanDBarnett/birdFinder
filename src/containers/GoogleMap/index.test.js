import React from 'react'; 
import { shallow } from 'enzyme';
import { GoogleMap } from '.';


describe('GoogleMap', () => { 
  it('should match the snapshot', () => { 
    const wrapper = shallow( <GoogleMap />);

    expect(wrapper).toMatchSnapshot(); 
  }); 
});