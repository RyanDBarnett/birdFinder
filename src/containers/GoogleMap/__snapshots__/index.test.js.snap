// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleMap should match the snapshot 1`] = `ShallowWrapper {}`;
